malloc/src/memgrind.c

```diff
@@ -1,6 +1,6 @@
 #include <stdlib.h>
 #include "mymalloc.h"
-
+#include "mymalloc.c" //wont compile without this
```

Comment:
i think it should compile if you run `./build.sh` and then run `malloc/build/main`
this would run the main executable specified in `CMakeLists.txt`, but I'm also still learning parts of this

once we do unit tests, CMake will also handle this and you can try running `ctest` after you `cd malloc/build` 😎